README.md

```diff
@@ -3,12 +3,26 @@
+|------------------------|-----------------------------------------------------|----------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `catalog.name`         | Catalog name                                        | true     | -             | "exampleCatalog"                                                                                                                                                                                           |
+| `namespace`            | Namespace                                           | true     | -             | "exampleNamespace"                                                                                                                                                                                         |
+| `table.name`           | Table name                                          | true     | -             | "exampleTable"                                                                                                                                                                                             |
+| `s3.endpoint`          | S3 endpoint URL                                     | true     | -             | "https://s3.example.com"                                                                                                                                                                                   |
+| `s3.accessKeyId`       | S3 Access Key ID                                    | true     | -             | "exampleAccessKeyId"                                                                                                                                                                                       |
+| `s3.secretAccessKey`   | S3 Secret Access Key                                | true     | -             | "exampleSecretKey"                                                                                                                                                                                         |
+| `s3.region`            | S3 region                                           | true     | -             | "us-east-1"                                                                                                                                                                                                |
+| `catalog.propertyName` | Set a catalog property with the name `propertyName` | true     | -             | {"catalog.uri": "http://localhost:8181"}                                                                                                                                                                   |
```

Comment:
could we also move this param to the end of the table, since it's the only "open" one.

Reply:
Done!